examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>Don't forget to remove before final version.</p><p> </p><p>Also, create a title for the notebook and an intro paragraph what this notebook is about.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>